### ISSUE TITLE
Rename package

### ISSUE THREAD

**Issue by HughParsonage** [thmcmahon/aes#1]
Name `aes` is not as descriptive as it could be. In particular, it connotes the AES algorithm for me. A longer but more evocative name might improve portability. 

Recommend: 

`library(AustElectionStudy)`

**Comment by thmcmahon** [comment]
Yeah I think this is a good idea - I need to email Mcallister and just check I'm OK on copyright grounds for this package and if I get that tick I'll change the name and try and submit it to CRAN.
